api/users.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt')
+const { JWT_SECRET } = process.env;
+const { createUser, getUser, getUserByUsername } = require('../db');
+const { requireUser } = require('./utils');
+
+
+// users/
+router.get('/', async (req, res, next) => {
+    const users = await getUser();
+    try {
+        res.send({
+            users
+        })
+    } catch (error) {
+        next(error)
+    }
+});
+
+// users/my account
+router.get('/myaccount', requireUser, async (req, res, next) => {
+            res.send({
+                ...req.user
+            })
+})
+
+// users/register
+router.post('/register', async (req, res, next) => {
+    const { username, password, name, email } = req.body
```

Comment:
we don't need the name here, I believe we decided not to include that in our users table for now